components/feature/push/src/main/java/mozilla/components/feature/push/Connection.kt

```diff
@@ -143,3 +189,24 @@ internal fun ServiceType.toBridgeType() = when (this) {
 
 @VisibleForTesting
 internal fun Protocol.asString() = name.toLowerCase(Locale.ROOT)
+
+internal typealias PushScope = String
+internal typealias AppServerKey = String
```

Comment:
Are these `typealiase` definitions valuable?

Reply:
They're internal to avoid confusion when switching between the different String values since they don't have types and were getting confusing during implementation.

Reply:
👍